pynicotine/gtkgui/userbrowse.py

```diff
@@ -603,11 +609,42 @@ def set_finished(self):
         self.refresh_button.set_sensitive(True)
         self.save_button.set_sensitive(not self.folder_tree_view.is_empty())
 
+    def populate_path_bar(self, folder_path=""):
+
+        for widget in list(self.path_bar):
+            self.path_bar.remove(widget)
+
+        for index, folder in enumerate(folder_path.split("\\")):
+            i_folder_path = "\\".join(folder_path.split("\\")[:index + 1])
+
+            if index:
+                label = Gtk.Label(label="/", visible=True)
```

Comment:
`"/"` -> `"\"`
Soulseek paths use backslashes.

Also, the horizontal padding seems a bit too spacious.

Reply:
I went with a backslash first, but @hboetes preferred a forward slash.

I can try reducing the padding slightly more, but I don't want to deviate from Nautilus' current design too much.

Reply:
In all other parts of the application, a backslash `\` is used in remote file paths, and on *nix a forward slash is used for local paths.

A forward slash here is technically incorrect on all systems, especially on Windows, where a forward slash is never seen in any path string.